Gemfile

```diff
@@ -37,7 +37,7 @@ group :test do
   gem "capybara"
   gem "selenium-webdriver"
   gem "simplecov", require: false
-  gem "webdrivers"
+  # gem "webdrivers" # removing this because it's causing issues with the CI -> https://github.com/titusfortner/webdrivers/issues/247#issuecomment-1641879165
```

Comment:
This appears to fix the failing tests issue.

See https://github.com/titusfortner/webdrivers/issues/247#issuecomment-1641139415
Failed CI job due to chromedriver issue: https://github.com/goodscary/firstrubyfriend/actions/runs/6385813400/job/17331346830